src/components/Message/Message.Bubble.tsx

```diff
@@ -121,24 +100,61 @@ export const Bubble = (props: Props, context: Context) => {
     </MessageBubbleTitle>
   ) : null
 
-  const bodyMarkup = body ? (
-    <MessageBubbleBody
-      className="c-MessageBubble__body"
-      isEmbed={isThemeEmbed}
-      dangerouslySetInnerHTML={{
-        __html: enhanceBody(body),
-      }}
-    />
-  ) : (
-    childrenMarkup
-  )
+  const hasOnlyOneChild = React.Children.count(children) === 1
+
+  const childrenMarkup = React.Children.map(children, child => {
```

Comment:
Nothing changed in the `childrenMarkup`, expect that I moved it closer to where it is called.